client/src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+const Button = styled.button`
+  background-image: linear-gradient(#fb8777, #ef1846);
+  cursor: pointer;
+  color: #f7efba;
+  border-radius: 10px;
+  box-shadow: 2px 4px 7px #aaaaaa;
+  border: none;
+  font-size: 14px;
```

Comment:
Why don't you use rem here for the font size? You can use this when you defined a font size in  globalstyles :)

Reply:
Yeees, i forgot that. Thank you 😊